test/expected/telemetry.out

```diff
@@ -395,6 +396,16 @@ SELECT t -> 'instance_metadata' FROM telemetry_report;
  {"cloud": "ci"}
 (1 row)
 
+-- Check access methods
+SELECT t->'access_methods' ? 'btree',
```

Comment:
Is there a specific reason why no actual numbers are included in the test? It might be good to check that not all values are 0.

Reply:
The reason is simply that they can vary between runs. It should be straightforward to check that they are greater than zero though.

Reply:
Switched to checking that they are greater than zero.